tssv/cli.py

```diff
@@ -31,6 +31,9 @@ def main():
     parser.add_argument(
         '-r', dest='report_handle', type=FileType('w'), default=stdout,
         help='name of the report file')
+    parser.add_argument(
+        '-f', dest='report_format', type=str, choices = ['text', 'json'],
```

Comment:
One last remark, perhaps it would be better to change the type of this option to `bool`, e.g., make the option '-j' for a JSON report and omit for a text report.